docs/content/concepts/cryptography/passkeys.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Passkeys
+description: Passkeys is a Sui primitive that enables you to sign-in to apps and send transactions to Sui using a private key securely stored on a authenticator. It uses the WebAuthn standard.
+---
```

Comment:
same as 
```
Sui supports passkey signature scheme that enables you to sign-in to apps and sign transactions to Sui using a private key securely stored on a passkey authenticator. It uses the WebAuthn standard.
```

(since passkey itself is not a sui primitive, sui just has ability to verify the signature produced by a passkey on a transaction now)